samples/tfm_integration/psa_level_1/README.rst

```diff
@@ -134,16 +134,19 @@ Build Zephyr with a non-secure configuration:
 
       $ west build -p -b lpcxpresso55s69_ns samples/tfm_integration/psa_level_1/ --
 
-Next we need to manually flash the secure (``tfm_s.hex``)
-and non-secure (``zephyr.hex``) images wth a J-Link as follows:
+Next we need to manually flash the resulting image (``tfm_merged.bin``) with a
+J-Link as follows:
 
    .. code-block:: console
 
       JLinkExe -device lpc55s69 -if swd -speed 2000 -autoconnect 1
-      J-Link>loadfile build/tfm/install/outputs/NXP/LPCXPRESSO55S69/tfm_s.hex
-      J-Link>loadfile build/zephyr/zephyr.hex
+      J-Link>loadfile build/tfm_merged.bin
```

Comment:
Same issue as earlier, perhaps mention the need to reset in some situations before we can flash.